types/v2/json.go

```diff
@@ -0,0 +1,93 @@
+package v2
+
+import (
+	"encoding/json"
+	"io"
+	"strconv"
+
+	"github.com/honeycombio/honeycomb-opentracing-proxy/types"
+)
+
+type ZipkinJSONSpan struct {
```

Comment:
Probably an useful reference: https://github.com/openzipkin/zipkin-api/blob/master/zipkin2-api.yaml#L511-L528

Reply:
This would be great as a comment above the type